src/Components/Server/src/Circuits/ArrayBuilderMemoryStream.cs

```diff
@@ -7,7 +7,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.AspNetCore.Components.Web.Rendering
+namespace Microsoft.AspNetCore.Components.Server.Circuits
```

Comment:
I did some misc cleanup of namespaces in the server project. There was this extra `.Web.Rendering` namespace hanging around without any real purpose.